templates/courant/mail.html.twig

```diff
@@ -1,147 +1,476 @@
+          background: #e59200;
+          border-style: solid;
+          border: none; 
+        }
+
+      u {
+          text-decoration: none;
+          border-bottom: 3px solid #e59200;
+      }
+
+      .grijs-donkerblauwLiOn {
+          background-image: -webkit-linear-gradient(95deg, #11273a 50%, #e8e8e8 50%);
+      }
+
+      .donker-mediumLiBo {
+        background-image: -webkit-linear-gradient(85deg, #043763 50%, #11273a 50%);
+      }
+
+      .medium-lichtLiOn {
+        background-image: -webkit-linear-gradient(95deg, #4e698b 50%, #043763 50%);
+      }
+
+      .licht-donkerLiBo {
+          background-image: -webkit-linear-gradient(85deg, #11273a 50%, #4e698b 50%);
+      }
+
+      .donker-lichtLiOn {
+        background-image: -webkit-linear-gradient(95deg, #4e698b 50%, #11273a 50%);
+      }
+
+      .licht-mediumLiBo {
+        background-image: -webkit-linear-gradient(85deg, #043763 50%, #4e698b 50%);
+      }
+
+      .inhoudKop {
+        text-align: center;
+        color: white;
+      }
+
+      .inhoudKolom {
+        width: 33%;
+        padding: 30px 10px 20px 10px;
+      }
+
+      li {
+        list-style: none;
+      }
+
+      .link {
+        text-decoration: none; 
+        text-align: center; 
+        color: white;
+      }
+
+      .liLink{
+        list-style: none; 
+        text-align: center; 
+        color: white; 
+        padding: 5px;
+      }
+
+    </style>
+  </head>
+  <body class="">
+      <tr>
+        <td>&nbsp;</td>
+        <td class="container">
+          <div class="content">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <table role="presentation" class="main">
+              <tr>
+                <td class="wrapper">
+          
+                  <!-- LOGO en TITEL -->                  
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="background: #e8e8e8">
+                    <tr style="height: 200px">
+                      <td id="logo" class="inhoud-lb">
+                        <img class="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/C.S.R._Delft_Logo.svg/190px-C.S.R._Delft_Logo.svg.png" style="width:150px;" />
+                      </td>
+                      <td style="text-align: center; vertical-align: middle;">
+                        <h1>De courant  </h1>
+                      </td>
+                    </tr>
+                  </table>
+
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+                    <tr > 
+                      <td class="grijs-donkerblauwLiOn">
+                        <div class="fowardslash">
+                          <hr>
+                        </div>
+                      </td>
+                    </tr>
+                  </table>
+
+                  <!-- INHOUD -->
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="background: #11273a">
```

Comment:
ARIA role presentation is hier niet op z'n plek, er staan zinnige dingen in deze table.

Ik zou me ook niet te druk maken over het toevoegen van aria roles.